1.1.c

```diff
@@ -0,0 +1,44 @@
+#include "tests.h"
```

Comment:
Testy są zbyt skromne jeśli chodzi o scenariusze, w dodatku nie weryfikujesz w nich tego, w jaki sposób użytkownik mógłby korzystać z Twojej biblioteki (nie weryfikujesz m.in. kluczowych rzeczy takich jak poprawność danych, zwracanie błędów przez funkcje). Napisałeś tzw. White Box testy, sprawdzasz wewnętrzną implementację biblioteki, ale nie sprawdzasz jej od strony potencjalnego użytkownika.